server/api/pipeline.go

```diff
@@ -442,3 +442,26 @@ var deleteStr = `[
 		"out": "logs purged by %s on %s\n"
 	}
 ]`
+
+func PostSkipStep(c *gin.Context) {
```

Comment:
step or workflow :D ...

please make a consistent naming acording to #745

Reply:
Okay.. which one do you think making more sense?

Reply:
well the issue is about workflows that are pending